app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php

```diff
@@ -109,7 +109,7 @@ protected function _buildMenuArray(?Varien_Simplexml_Element $parent = null, $pa
 
         uasort($parentArr, [$this, '_sortMenu']);
 
-        foreach ($parentArr as $key => $value) {
+        foreach (array_keys($parentArr) as $key) {
             $last = $key;
         }
         if (isset($last)) {
```

Comment:
```suggestion
        if ($parentArr !== []) {
            $last = array_key_last($parentArr);
```

Reply:
Nice idea! Let me check.

Reply:
There is another rule for array_key_last that can be extended to cover that ...

https://github.com/rectorphp/rector/blob/main/rules/Php73/Rector/FuncCall/ArrayKeyFirstLastRector.php

Reply:
`array_key_last`  returns null for an empty array ... so it should be enought to check that `$last` is nut null.